frontend/src/pages/Landing/Footer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { ReactElement } from 'react';
+import styles from './landing.module.css';
+import { dti_logo } from '../../icons/other';
+
+const Footer = () => {
+  return (
+    <div className={styles.footer}>
+      <div className={styles.blackTop}>
+        <p
+          style={{
+            color: 'white',
+            fontSize: 25,
+            textAlign: 'left',
+            paddingLeft: 100,
+            paddingTop: 30,
+            paddingBottom: 30,
+          }}
+        >
+          Students schedule, <br />
+          edit, and cancel rides <br />
+          on rider web <br />
+          and mobile.
+        </p>
+        <img src="frontend/src/pages/Landing/Group525.png" />
```

Comment:
I think this should be changed to have the src use the imported image from the suggestion above.